### PR TITLE
Count numbers Added

### PR DESCRIPTION
**Pull request by berkkant**
I have enhanced the api by adding count numbers of followers, followings, and art items of a specific user.

### CODE REVIEW

File: art-community-platform/backend/app/api/views/user.py [hunk 1]
```diff
@@ -68,10 +68,20 @@ def get_user_by_id(req, user_id):
     if user_id in request_sender.followings:
         is_following = True
 
+    followers = get_followers_by_user_id_helper(user_id=user_id)
+    followings = get_followings_by_user_id_helper(user_id=user_id)
+    art_item_list = get_art_items_by_user_id_helper(user_id=user_id)
```
Comment (burakaktan) [inline art-community-platform/backend/app/api/views/user.py:73]: Is there a need for this 3 lines?

**Review by burakaktan — APPROVED** [review]
There are 3 lines which are not used.
Other than that, the issue is good!

File: art-community-platform/backend/app/api/views/user.py (hunk 1 above)
Comment (burakaktan) [reply art-community-platform/backend/app/api/views/user.py:73]: They are used in lines 75,76,77. Sorry!

Comment (berkkant) [reply art-community-platform/backend/app/api/views/user.py:73]: Yes, I call for helper functions to get follower/following/art-item list. After that I get their counts.

**Pull request merged**